apps/src/appMain.js

```diff
@@ -62,6 +62,13 @@ export default function (app, levels, options) {
 
   studioApp().configure(options);
 
+  if (options.level && options.level.shapeShift) {
+    const rotate = m => m[0].map((x, i) => m.map(x => x[m.length - i - 1]));
+    for (let i = 0; i < Math.random() * 4; i++) {
```

Comment:
Good catch!  I totally missed that `Math.random()` will regenerate on each iteration.